src/main/java/de/tum/in/www1/artemis/service/LearningObjectService.java

```diff
@@ -63,11 +68,29 @@ public LearningObjectService(ParticipantScoreService participantScoreService, Ex
     public boolean isCompletedByUser(@NotNull LearningObject learningObject, @NotNull User user) {
         return switch (learningObject) {
             case LectureUnit lectureUnit -> lectureUnit.isCompletedFor(user);
-            case Exercise exercise -> participantScoreService.getStudentAndTeamParticipations(user, Set.of(exercise)).anyMatch(score -> score.getLastScore() >= MIN_SCORE_GREEN);
+            case Exercise exercise -> isCompletedByUser(exercise, user);
             default -> throw new IllegalArgumentException("Learning object must be either LectureUnit or Exercise");
         };
     }
 
+    /**
+     * Checks if the user has completed the exercise.
+     * For manually assessed exercises the user cannot get more feedback usually, so we have to assume all submissions as completed for now
+     * For other exercises the user can get more assessments and achieve high enough scores to surpass the {@link MIN_SCORE_GREEN} threshold
+     *
+     * @param exercise the exercise
+     * @param user     the user for which to check the completion status
+     * @return true if the user has completed the exercise
+     */
+    private boolean isCompletedByUser(Exercise exercise, User user) {
+        if (exercise.getAssessmentType() == AssessmentType.MANUAL) {
+            return submissionRepository.existsByExerciseIdAndParticipantIdAndSubmitted(exercise.getId(), user.getId());
+        }
+        else {
+            return participantScoreService.getStudentAndTeamParticipations(user, Set.of(exercise)).anyMatch(score -> score.getLastScore() >= MIN_SCORE_GREEN);
```

Comment:
Does it make sense to offer generic or auto-generated feedback even for manual assessments? 
What is the `MIN_SCORE_GREEN` threshold for which no feedback is shown (I suppose as for the comments)?

Reply:
This does not affect the feedback provided for the student. This method helps the competencies and learning paths to determine if the exercise is completed or should be shown to the student as still open. 
Manually assessed exercises can only be reasonably solved by a student once, since any submission afterwards will probably not get assessed. Automatically assessed exercises on the other hand can be repeated over and over. We aim for at least 80% (MIN_SCORE_GREEN)

Reply:
Ah okay good to know, then never mind 👍